database/seeders/TagSeeder.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Database\Seeders;
+
+use App\Models\Tag;
+use Illuminate\Database\Seeder;
+
+class TagSeeder extends Seeder
```

Comment:
Where we use this seeder? I think that we can add it do database seeder or demo seeder when pr https://github.com/blumilksoftware/interns2025c/pull/26 will be merged before this